opal/Makefile.am

```diff
@@ -25,6 +25,7 @@ SUBDIRS = \
         etc \
         util \
 	mca/base \
+	mca/common/verbs \
```

Comment:
This line should absolutely not be here.  It is causing "make distclean" to fail.